src/InputBox.css

```diff
@@ -0,0 +1,120 @@
+::-webkit-scrollbar {
+  display: none;
+}
+
+.input-container {
+  width: 95%;
+  max-width: 600px;
+  height: fit-content;
+  border: 1px solid #5f6368;
+  border-radius: 8px;
+  box-shadow: 0 15px 30px rgb(0 0 0 / 65%);
+  padding: 1rem;
+  padding-top: 0px;
+  display: flex;
+  flex-direction: column;
+}
+.input-container > input[type="text"] {
+  width: 100% !important;
+  font-family: "Google Sans", Roboto, Arial, sans-serif;
+  font-weight: 500;
+  font-size: 1.1rem;
+  color: white;
+  padding: 10px 0px;
+  border: none;
+  background-color: transparent;
+}
+input[type="text"]:focus-visible {
+  outline: none !important  ;
+}
+textarea {
+  width: 100% !important;
+  height: 100px !important;
+
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 0.875rem !important;
```

Comment:
Avoid using important at so many places. Rather create a more specific class and then apply it.